vsdkx/addon/zoning/processor.py

```diff
@@ -38,14 +38,20 @@ def pre_process(self, addon_object: AddonObject) -> AddonObject:
             (AddonObject): addon object has updated information for frame,
             inference, result and/or shared information:
         """
-        for i in range(len(self._remove_areas)):
-            xmin = self._remove_areas[i][0]
-            ymin = self._remove_areas[i][1]
-            xmax = self._remove_areas[i][2]
-            ymax = self._remove_areas[i][3]
-            addon_object.frame[ymin:ymax, xmin:xmax] = cv2.blur(
-                addon_object.frame[ymin:ymax, xmin:xmax],
-                (30, 30)
+        for area in self._remove_areas:
+            roi_corners = np.array(
+                [area],
+                dtype=np.int32)
+            blurred_image = cv2.GaussianBlur(addon_object.frame, (53, 53), 30)
+            mask = np.zeros(addon_object.frame.shape, dtype=np.uint8)
+            channel_count = addon_object.frame.shape[2]
+            ignore_mask_color = (255,) * channel_count
```

Comment:
Same as above, create a constant for `color_white = 255`

Reply:
I can't see any practical implication of storing something like this in a variable. Why would you want to complicate your code by storing stuff in a variable that you would never want to change, or do you see a scenario when we would want to configure a color of a mask?!

number 255 itself is not a white color itself, it becomes white when multiplied by `channel_count` so it can be potentially misleading as well.

Reply:
I think it's an overkill

Reply:
for code readability, instead of having the same hardcoded numbers being repeated in the code. And in some cases this is best practice since you have one central spot where you can simply update this variable.

Reply:
About this point: `number 255 itself is not a white color itself, it becomes white when multiplied by channel_count so it can be potentially misleading as well.` You can name it whatever your heart desires, the naming was just a suggestion.